crates/indexer/src/geyser/accounts/hpl_reward_center/purchase_ticket.rs

```diff
@@ -187,3 +234,27 @@ pub(crate) async fn process(
 
     Ok(())
 }
+
+fn calculate_payout(price: u64, rule: &RewardRule) -> Result<(BigDecimal, BigDecimal)> {
```

Comment:
so good @imabdulbasit. ty.

Reply:
A note that we can look at exporting this method form the program crate so it stays in sync.